### DISCUSSION TITLE
Retrieving specific objects from a glb model converted with IfcConvert

### DISCUSSION THREAD

**Discussion by Yusuke-Togao** [IfcOpenShell/IfcOpenShell#5430] [Q&A]
I have converted the entire model of an ifc file to a glb file with IfcConvert. Is there a way to retrieve specific objects from this?

For example, can I find a mesh or group corresponding to the GlobalID that an object had when it was an ifc file, in a glb model loaded onto the scene with GLTFLoader?

**Comment by aothms** [comment]
The nodes in gltf have names like `product-145b3b0a-bf19-4509-ba9a-e1604ce9fa67-body`. The first part always product- because not all fileformats support leading numbers as an identifier. And the hexadecimal encoding of the guid. `from ifcopenshell import guid; g.split(g.expand(inst.GlobalId))` because not  all fileformats allow all ifc base64 characters (`$` in particular) in identifiers. If you use `IfcConvert --use-element-guids` you get the GlobalId identifiers in the output directly (which is invalid for collada, but should work for your use case). `-body` is the representation context identifier. If you run with `IfcConvert --model --plan` you'll also get `-axis` or `-footprint` representations.
